lte/protos/mconfig/mconfigs.proto

```diff
@@ -72,6 +72,61 @@ message EnodebD {
         int32 tac = 8;
         int32 cell_id = 9;
         string ip_address = 10;
+        // EnodeB Device Neighbor Freq parameters.
+        map <string, neighbor_freq_table> neighbor_freq_list = 11;
+        // EnodeB Device Neighbor Cell parameters.
+        // https://usp-data-models.broadband-forum.org/tr-196-2-1-1-usp.html#D.
+        // FAPService:2.FAPService.{i}.CellConfig.UMTS.RAN.NeighborList.
+        map <string, neighbor_cell_table> neighbor_cell_list = 12;
```

Comment:
This looks great! Clear documentation in proto definitions and swagger definitions are two things I believe will contribute towards long term code health.